src/pages/admin/Memberships/Memberships.js

```diff
@@ -0,0 +1,204 @@
+
+  useEffect(() => {
+    let isSubscribed = true;
+    const getMemberships = async () => {
+      const res = await fetchBackend(`/memberships`, "GET");
+      if (isSubscribed) {
+        setMembershipData(res);
+      }
+    };
+    getMemberships();
+
+    return () => (isSubscribed = false);
+  }, []);
+
+  return (
+    <div>
+      <Helmet>
+        <title>BizTech Memberships</title>
+      </Helmet>
+      <div className={classes.container}>
+        {/* Left panel for additional event filters (only on desktop view) */}
+        {!isMobile && (
+          <div className={classes.sidePanelLayout}>
+            <Typography variant="h1">Memberships</Typography>
+            <List>
+              {Object.values(PERSONALIZATION_STATES).map((pState) => (
+                <ListItem
+                  key={pState.index}
+                  className={
+                    personalizationIndex === pState.index
+                      ? classes.sidePanelActiveButton
+                      : classes.sidePanelButton
+                  }
+                  onClick={() => handlePersonalizationChange(pState.index)}
+                  button
+                >
+                  <ListItemText>
+                    {pState.icon}&nbsp;{pState.displayName}
+                  </ListItemText>
+                </ListItem>
+              ))}
+            </List>
+          </div>
+        )}
+        <div className={classes.tabsLayout}>
+          {/* Upper tabs for filtering (only on desktop view) and searching for events */}
+          <div className={classes.tabsContainer}></div>
+          {/* Filters in mobile view */}
+          {isMobile && (
+            <div className={classes.mobileFilters}>
+              {Object.values(PERSONALIZATION_STATES).map(
+                (pState) =>
+                  pState.displayName !== "Individual" && // don't render "All"
+                  (personalizationIndex === pState.index ? (
+                    <Chip
+                      key={pState.displayName}
+                      className={classes.chipFilter}
+                      size="small"
+                      color="primary"
+                      label={pState.displayName}
+                      onDelete={() =>
+                        handlePersonalizationChange(
+                          PERSONALIZATION_STATES.INDIVIDUAL.index
+                        )
+                      }
+                    />
+                  ) : (
+                    <Chip
+                      key={pState.displayName}
+                      className={classes.chipFilter}
+                      size="small"
+                      color="secondary"
+                      label={pState.displayName}
+                      onClick={() => handlePersonalizationChange(pState.index)}
+                    />
+                  ))
+              )}
+            </div>
+          )}
+
+          {personalizationIndex === 0 && (
```

Comment:
can we just use the keys of PERSONALIZATION_STATES instead of indices? that way we don't need indices at all. For example, personalizationState === "SUMMARY". 

I don't think the current code is very scalable because a new index and a need {personalizationIndex === X && <SomeComponent /> would have to be added every time a new tab were to be added. Maybe PERSONALIZATION STATES could look like: 
`const PERSONALIZATION_STATES = {
  SUMMARY: {
    displayName: "Summary",
    icon: <StarBorderIcon fontSize="small" />,
    component <Summary />
  },
  QUESTION: {
    displayName: "Question",
    icon: <HelpOutlineIcon fontSize="small" />,
    component: <Questions />
  },
  INDIVIDUAL: {
    displayName: "Individual",
    icon: <ListIcon fontSize="small" />,
    component: <Individual />
  },
};`

and the code would simply show whichever the current state is and whatever component corresponds with it. So maybe like {currState.component}